org.eclipse.emf.compare-parent/pom.xml

```diff
@@ -40,25 +40,25 @@
 
     <!-- plugins versions -->
     <!-- use 'mvn versions:display-plugin-updates' to check for the latest -->
-    <tycho-version>2.1.0</tycho-version>
-    <tycho-extras-version>2.1.0</tycho-extras-version>
-    <maven-resources-version>2.6</maven-resources-version>
-    <maven-antrun-version>1.7</maven-antrun-version>
-    <findbugs-maven-version>3.0.0</findbugs-maven-version>
-    <maven-pmd-version>3.1</maven-pmd-version>
+    <tycho-version>3.0.5</tycho-version>
```

Comment:
The latest released version of Tycho is 4.0.4.
Is there a reason to not update to even that?

Reply:
The point was to try and limit the breaking changes, since updating tycho version is usally a painful thing. I'm planning on searching for the least amount of modification that makes this work again.

Reply:
Sure. It was mainly a suggestion.
Often the update the update is trivial, so maybe you just want to try it out. :)